packages/web/src/utils/stake-position-utils.ts

```diff
@@ -80,6 +82,62 @@ export const convertToKMB = (
   }
 };
 
+export const convertLiquidityUsdToKMB = (
+  price: string,
+  maximumFractionDigits?: number,
+  minimumFractionDigits?: number,
+  prefix?: string,
+) => {
```

Comment:
What do you think about taking input as an option object?

In the example usage, 
it looks unnatural to arbitrarily put `undefined`.
```
convertLiquidityUsdToKMB(position.positionUsdValue, undefined, undefined, “$”);
```

I think the following looks better.
```
convertLiquidityUsdToKMB(position.positionUsdValue, {
  prefix: "$",
});
```

Reply:
i will fix this